src/client/components/ModelViewRegisteringForAnEvent/ModelViewRegisteringForAnEvent.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { AppHeader } from '../Appheader/AppHeader.component';
+import InputLink from '../Input/Input';
+import Dropdown from '../Dropdown/DropDown';
+import { Button } from '../Button/Button';
+import { Card } from '../Card/Card';
+import Footer from '../footer/footer';
+import Logo from './HYF.logo.svg';
+import Modal from '../Modal/Modal';
+import './ModelViewRegisteringForAnEvent.css';
+import { text } from '@storybook/addon-knobs';
+
+const questions = [
+  {
+    id: '01',
+    value: 'Add Question here',
+  },
+  {
+    id: '02',
+    value: 'question2',
+  },
+  {
+    id: '03',
+    value: 'question3',
+  },
+];
+
+const ModelViewRegisteringForAnEvent = () => {
+  const [value, setValue] = useState('');
+
+  const submit = () => {
+    return 'submit';
+  };
+
+  const canContinue = !!value;
+  return (
+    <div className="wrapper">
+      <div className="header">
+        <AppHeader path={Logo} />
+      </div>
+      <div className="model">
+        <Modal title="Register your spot">
+          <div className="input dropDown">
+            <Dropdown
+              items={questions}
+              value={value}
+              setValue={setValue}
+              require
+            />
+          </div>
+          <div className="input link">
+            <InputLink />
```

Comment:
Please keep the name `Input` since this is a text to be entered.
Also, use the `value, onChange` from the `Input` and put it in the local state.